Distiller-server/src/main/java/com/mapr/distiller/server/Coordinator.java

```diff
@@ -2115,19 +2116,70 @@ public void run() {
 							+ " as it was disabled after retrieval from schedule.");
 				}
 			}
-
-			/*
-			 * if (System.currentTimeMillis() >= lastStatus + statusInterval) {
-			 * lastStatus = System.currentTimeMillis();
-			 * LOG.debug("Main: Printing status " + lastStatus); String
-			 * recordProducerStatus = getRecordProducerStatus();
-			 * 
-			 * LOG.debug("Record Queues"); String recordQueues =
-			 * getRecordQueues();
-			 * 
-			 * LOG.debug("Metric Actions"); String metricActions =
-			 * getMetricActions(); }
-			 */
+			/**
+			 * Some status stuff that is useful in debugging
+			if(System.currentTimeMillis() >= lastStatus + statusInterval){
```

Comment:
No, we don't need it there.  But I put it there for the time being so when I need to make changes in Coordinator I can get a clean diff without needing to cut the content from my local file, commit and push, then paste back into my local file before each checkin.